pkg/exchange/okex/query_closed_orders_test.go

```diff
@@ -0,0 +1,62 @@
+package okex
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/testutil"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_QueryClosedOrders(t *testing.T) {
+
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, types.ExchangeOKEx.String())
+	if !ok {
+		t.Skip("Please configure all credentials about OKEX")
+	}
+
+	e := New(key, secret, passphrase)
+
+	queryOrder := types.OrderQuery{
+		Symbol: "BTCUSDT",
+	}
+
+	// test by order id as a cursor
+	closedOrder, err := e.QueryClosedOrders(context.Background(), string(queryOrder.Symbol), time.Time{}, time.Time{}, 609869603774656544)
```

Comment:
I guess this order ID is your own order and which can work for others

Reply:
Yes, I will comment out the test